test/product-import.spec.coffee

```diff
@@ -510,6 +510,19 @@ describe 'ProductImport unit tests', ->
         done()
       .catch done
 
+    it 'should throw error in case of too big query', (done) ->
+      exptectedErrorMessage = 'product fetch query size: 10952 bytes, exceeded the supported size, please try with a smaller batch size.'
```

Comment:
@hajoeichler sounds like a nice idea... I will look into it when I am solving the issue: https://github.com/sphereio/sphere-product-import/issues/5

right now we need an urgent solution for a project, where problem can be solved by adjusting the batch size. Hence the pull request.